src/styles/blocks/header.scss

```diff
@@ -0,0 +1,21 @@
+@import "../utils/normalize";
+@import "../utils/reset";
+
+.header {
```

Comment:
why do you import sheets on each file, I think it`s redundant - all of them on 'main.scss' can be enough

Reply:
In one of videos dedicated to SASS topic Misha said that utils shoud be linked to the file where they'll be used, not to the main file.